public/scripts/search.js

```diff
@@ -1,12 +1,12 @@
-import { ORDER_RELEVANCE } from './ordering.js';
-import { debounce } from './utils.js';
-import { Searcher } from 'fast-fuzzy';
+import {Searcher} from 'fast-fuzzy';
+import {ORDER_RELEVANCE} from './ordering.js';
+import {debounce} from './utils.js';
 
 const QUERY_PARAMETER = 'q';
 
 const getQueryFromParameter = (location, parameter) => {
   const query = new URLSearchParams(location.search).get(parameter);
-  return query ? query : '';
+  return query ?? '';
```

Comment:
What about this?

```suggestion
  return query || '';
```

The double question mark looks really unintuitive to me comparing against a default.